.github/workflows/update-reference.yml

```diff
@@ -6,21 +6,59 @@ on:
     - cron: "0 8 * * 1" # At 08:00 on Monday
 
 jobs:
+  generate-schema:
+    runs-on: ubuntu-latest
+    container: python:3.9
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          repository: saleor/saleor
+          ref: main
+
+      - name: Install system dependencies
+        run: apt-get install -y libpq-dev
+
+      - name: Install dependencies
+        run: |
+          python -m pip install wheel
+          python -m pip install -r requirements_dev.txt
+
+      - name: Generate schema
+        run: python manage.py get_graphql_schema > schema.graphql
+
+      - name: Upload schema artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: schema
+          path: schema.graphql
+
```

Comment:
We should not do that and, instead, add the schema file to published assets for every core release on GitHub.

Reply:
OK, what's the ETA for that?

Reply:
We need to look into modifying our release script to include that, no ETA yet as it wasn't planned.